api/_api-service.js

```diff
@@ -7,33 +7,34 @@ module.exports = {
     try {
       const key = process.env.goodreads_key;
       const goodreadsResponseXML = await got(
-          `https://www.goodreads.com/review/list/92378107.xml?key=${key}&v=2&shelf=read`,
+        `https://www.goodreads.com/review/list/92378107.xml?key=${key}&v=2&shelf=read`
       );
 
       goodreadsResponseJSON = convert.xml2json(goodreadsResponseXML.body, {
         compact: true
       });
 
-      const reviews = JSON.parse(goodreadsResponseJSON).GoodreadsResponse.reviews.review;
+      const reviews = JSON.parse(goodreadsResponseJSON).GoodreadsResponse
```

Comment:
I'm not very happy with how prettier has moved a part of this statement to the next line. It compromises readability.  I have a suggestion, let's keep the parsed `goodreadsResponseJSON` in a different variable and then extract out it's properties.

Reply:
After running prettier, it is now on the same line. Don't know how it is on the next line on this PR

Reply:
Let's either keep the whole statement on a new line or store the parsed value in a new variable so as to work around this issue.